po/da.po

```diff
@@ -718,29 +716,27 @@ msgstr ""
 
 #: src/SUMMARY.md:221
 msgid "embedded-hal"
-msgstr ""
+msgstr "embedded-hal"
 
 #: src/SUMMARY.md:222
 msgid "probe-rs, cargo-embed"
-msgstr ""
+msgstr "probe-rs, cargo-embed"
 
 #: src/SUMMARY.md:223
 msgid "Debugging"
-msgstr ""
+msgstr "Debugning"
```

Comment:
nit: Haven't come across this word. Other options (not necessarily better):
- aflusning (old school)
- fejlfinding

Reply:
Oh, "fejlfinding" i much better, thanks! :smile: